UnityProject/Assets/Scripts/Inventory/NamedSlot.cs

```diff
@@ -43,11 +43,12 @@ public enum NamedSlot
 [Serializable, Flags]
 public enum NamedSlotFlagged
 {
-None = 1 << 0,
-Outerwear = 1 << 1,
-Belt = 1 << 2,
-Head = 1 << 3,
-Feet = 1 << 4,
+None = 1 << -1,
```

Comment:
why -1 ?

Reply:
Because the equivalent `NamedSlot` is `2048` which would conflict with one of the `NamedSlotFlagged` slots.

Reply:
Probably shouldn't bit shift with a negative number. What do you suggest?

Reply:
Why don't you put the number in exactly 
None = 2048
?

Reply:
Won't that conflict with `NamedSlotFlagged.Hands` (1 << 11 => 2048)? I swapped to assigning the number with bitshift instead of a non-shifted number to make it more readable, as per the suggestion in PR #5130.

Reply:
I'm confused why you can't just have zero?

Reply:
Ah, of course. Done.